### PR TITLE
Created 2 versions of product views in product category view and posi…

### PR DESCRIPTION
**Pull request by mSobonVirtua**
…billity switching between them

### CODE REVIEW

File: my-project/templates/product_category/_product_list.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<table class="table table-hover table-striped">
```
Comment (astyczen) [inline my-project/templates/product_category/_product_list.html.twig]: tables should be used to display content as its a very old approach, use div's

File: my-project/templates/product_category/_product_grid.html.twig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            <div class="col-4 text-center product-template">
+                <div>TODO ADD PRODUCT IMG</div>
+                <div>{{ product.name }}</div>
+                <div>TODO ADD PRODUCT PRICE</div>
+                <a class="btn btn-success" href="{{ path('WishListNew', {'productId': product.id}) }}">Add to WishList</a>
+            </div>
+        </a>
+    {% endfor %}
+</div>
+<style>
```
Comment (astyczen) [inline my-project/templates/product_category/_product_grid.html.twig]: lets move styles to a separate file

File: my-project/templates/product_category/show.html.twig [hunk 3]
```diff
@@ -45,17 +45,25 @@
...
-                    </td>
-                {% endfor %}
-            </tr>
         </tbody>
     </table>
+    <h3 class="text-center">Products in this category
+        <span>
+            {% if viewType == "list" %}
+                <a class="btn btn-primary" href="{{ path("product_category_show", { 'id': product_category.id }) }}?viewType=grid">Show grid</a>
+            {% endif %}
```
Comment (astyczen) [inline my-project/templates/product_category/show.html.twig]: lets use else, or just use if == grid near vew type and link content

File: my-project/templates/product_category/show.html.twig [hunk 4]
```diff
@@ -45,17 +45,25 @@
...
+            {% if viewType == "grid" %}
+                <a class="btn btn-primary" href="{{ path("product_category_show", { 'id': product_category.id }) }}"> Show list</a>
+            {% endif %}
+        </span>
+    </h3>
+    {% if viewType == "list" %}
+        {{ include("product_category/_product_list.html.twig") }}
+    {% endif %}
```
Comment (astyczen) [inline my-project/templates/product_category/show.html.twig]: lets use else

File: my-project/templates/product_category/_product_list.html.twig (hunk 1 above)
Comment (mSobonVirtua) [reply my-project/templates/product_category/_product_list.html.twig]: @astyczen But bootstrap styles need `<table>` tag

Comment (astyczen) [reply my-project/templates/product_category/_product_list.html.twig]: we should copy those styles into our file and adjust them to use classes

**Review by astyczen — APPROVED** [review]

**Pull request merged**